.github/ISSUE_TEMPLATE/zpa_epic.md

```diff
@@ -0,0 +1,80 @@
+---
+
+name: Zeebe Process Automation team Epic
+about: Prepare a new Epic for the ZPA team
+title: ''
```

Comment:
```suggestion
title: '[EPIC] '
```

❓ Not sure if we would want to do this, but so far, our epic issues are often prefixed with something like this. What do you think?